src/test/component.test.js

```diff
@@ -0,0 +1,111 @@
+import { render, screen } from '@testing-library/react';
+import Mycalculator from '../component/Calculatore';
+import Home from '../component/home';
+import { Qutes } from '../component/CalculatoreButtons';
+
+describe('Test Calculatore Component', () => {
+  test('Test text on Calculator Component', () => {
+    render(<Mycalculator />);
+    const pElement = screen.getByText(/Letus do same think new/i);
+    expect(pElement).toMatchSnapshot();
+  });
+  test('Test AC btn on calculatore', () => {
+    render(<Mycalculator />);
+    const pElement = screen.getByText(/AC/i);
+    expect(pElement).toMatchSnapshot();
+  });
+
+  test('Test % btn on calculatore', () => {
+    render(<Mycalculator />);
+    const pElement = screen.getByText(/%/i);
```

Comment:
- [ ] All your tests are in this file on point but there is a small issue. What component test meant in the requirement is that you create different files for different components. for example for your component `Calculatore.js` create a test file names `calculatore.test.js`, for home component, create a test file named `home.test.js` and so on and so fourth. 
by convention we don't test all our components in one file, we actually create different test files for our different components. kindly consider implement this change in organizing your test files.

Reply:
🙏🙏🙏 Thank you very much for your good and constructive comment. we will do it according your comment.